app/templates/events/event_list.html

```diff
@@ -104,9 +104,66 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
               </td>
             {% endif %}
           </tr>
+          {% endif %}
         {% endfor %}
      </tbody>
    </table>
+   {% if user.isCeltsStudentStaff or user.isAdmin %}
+   <button class="toggle-table text-primary border border-primary border-2">Past Events</button>
+ <table class="table d-none">
```

Comment:
Make sure your indentation levels are matching and following what comes before it.